src/test/java/com/wikia/webdriver/common/users/CreateUser.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.common.users;
+
+import org.joda.time.DateTime;
+
+public class CreateUser {
+  private String userName = "User" + DateTime.now().getMillis();
+  private String password = "Pass" + DateTime.now().getMillis();
+  private String email = "qaart001+" + DateTime.now().getMillis() + "@gmail.com";
+  private DateTime birthday = new DateTime(1952, 12, 12, 12, 0, 0);
+
+  public CreateUser() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)